contracts/deploy/mainnet/106_ousd_metamorpho_usdc.js

```diff
@@ -0,0 +1,98 @@
+const addresses = require("../../utils/addresses");
+const { deploymentWithGovernanceProposal } = require("../../utils/deploy");
+
+module.exports = deploymentWithGovernanceProposal(
+  {
+    deployName: "106_ousd_metamorpho_usdc",
+    forceDeploy: false,
+    // forceSkip: true,
+    // reduceQueueTime: true,
+    deployerIsProposer: true,
+    proposalId:
+      "104829708818146043512994516146479504527533094502750752547120869250660459466838",
```

Comment:
This seems like a wrong proposal ID

Reply:
My bad, I forgot to change it. It should be good now.